qiita_pet/templates/private_studies.html

```diff
@@ -3,13 +3,18 @@
 <link rel="stylesheet" href="/static/vendor/css/jquery.dataTables.css" type="text/css">
 
 <script src="/static/vendor/js/jquery.dataTables.min.js"></script>
+<script src="/static/vendor/js/jquery.dataTables.plugin.natural.js"></script>
 
 <script type="text/javascript">
 var current_study;
 
 $(document).ready(function() {
-    $('#user-studies-table').dataTable({"order": [[ 1, "asc" ]]});
-    $('#shared-studies-table').dataTable({"order": [[ 1, "asc" ]]});
+        $('#user-studies-table').dataTable({
+            order: [[ 1, "asc" ]],
+            columnDefs: 'natural'});
+        $('#shared-studies-table').dataTable({
+            order: [[ 1, "asc" ]],
+            columnDefs: 'natural'});
```

Comment:
Nope, I don't think it works see screenshot below of studies sorted by number of samples ascending):

![image](https://cloud.githubusercontent.com/assets/2574407/6274823/7afa6694-b837-11e4-84b2-2fd555346bda.png)

Note that the problem only presents when the column is a mixed datatype (integers and strings in the column, as is the case when there are some studies that have `num_samples_collected` set to a number, and some studies that do not have it set (which displays as `None`)